classes/production/laker-polling/application.groovy

```diff
@@ -0,0 +1,61 @@
+server {
```

Comment:
Delete this file from your repository. It's in the git ignore. but since it's on the repo, it's still part of the merge.